plugins/modules/azure_rm_search.py

```diff
@@ -0,0 +1,490 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 David Duque Hernández, (@next-davidduquehernandez)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
Please remove the METADATA block it is no longer needed!

Reply:
Done!